index.html

```diff
@@ -11,7 +11,7 @@
 </head>
```

Comment:
- [ ] [OPTIONAL] In line 7 modify the title tag from `<title><a href="">GintsMisins Portfolio</a></title>` to `<title>GintsMisins Portfolio</title>` because now your title is showing like this in the browser tab 👍🏼 
![image](https://user-images.githubusercontent.com/43698511/133910229-2e9b6288-8b83-4581-8989-1dc536b288f5.png)